examples/debouncing/src/main/scala/example/Main.scala

```diff
@@ -0,0 +1,67 @@
+package example
+
+import cats.effect.IO
+import tyrian.Html.*
+import tyrian.*
+
+import scala.concurrent.duration.*
+import scala.scalajs.js.annotation.*
+
+@JSExportTopLevel("TyrianApp")
+object Main extends TyrianIOApp[Msg, Model]:
+
+  val DEBOUNCING_MILLIS: Int = 500
+  val TICK_INTERVAL: Int = 100
```

Comment:
Stylistically in Scala it's unusual to see this sort of formatting. Sometimes people capitalise constants, so perhaps consider:

```scala
  val DebouncingMillis: Int = 500
  val TickInterval: Int     = 100
```

Reply:
Done. And this time I remembered to run the formatter